terra_layer/migrations/0041_auto_20191003_1701.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 2.1.13 on 2019-10-03 15:01
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def forward(apps, schema_editor):
+    Scene = apps.get_model('terra_layer', 'Scene')
+
+    try:
+        for slug, data in settings.TERRA_LAYER_VIEWS.items():
+            Scene.objects.create(
+                slug=slug,
+                name=data['name'],
+            )
+    except AttributeError:
+        print("No default layers' views was found. Don't forget to add some.")
```

Comment:
Could you use logger.info instead or a print ?

Reply:
Done